### DISCUSSION TITLE
check the download by CHECKSUM

### DISCUSSION THREAD

**Discussion by frazucca** [CS-SI/eodag#1405] [General]
I wonder if any good practices of verify the download by the way of _Checksum_ it's available for **eodag,** especially for the different CDSE providers.
 
Any hints it's welcomed
TIA

ciao
franz

**Comment by sbrunato** [comment]
Hello @frazucca , unfortunately checksum is not is not retrieved from metadata in `cop_dataspace`. But we can plan to add this as this is returned by the provider.
This is not automatically returned in product properties because this metadata is appart of other properties in `cop_dataspace` response.
